plugins/cpu/fu-plugin-cpu.c

```diff
@@ -73,6 +75,18 @@ fu_plugin_add_security_attrs_intel_cet (FuPlugin *plugin, FuSecurityAttrs *attrs
 		return;
 	}
 
+	/* check that userspace has been compiled for CET support */
+	if (!g_spawn_command_line_sync (FWUPD_LIBEXECDIR "/fwupd/fwupd-detect-cet",
+					NULL, NULL, &exit_status, &error_local)) {
+		g_warning ("failed to test CET: %s", error_local->message);
```

Comment:
Shouldn't this be a failure or unused too?

Reply:
I wasn't sure. This is more of a "distributor of fwupd f*cked up" situation rather than anything the hardware vendor did.

Reply:
Well I think it's missing a `return` then.  Right now if that fails to run, it will end up with a success attribute and enabled which is incorrect.

Reply:
Right, agreed, fixed. Thanks.